en_ewt-ud-dev.conllu

```diff
@@ -14230,7 +14230,7 @@
 13	DM	dm	VERB	VB	VerbForm=Inf	11	xcomp	11:xcomp	SpaceAfter=No
 14	...	...	PUNCT	,	_	11	punct	11:punct	SpaceAfter=No
 15	although	although	SCONJ	IN	_	18	mark	18:mark	_
-16	DM's	dm'	NOUN	NNS	Number=Plur	18	nsubj	18:nsubj	_
+16	DM's	dm	NOUN	NNS	Number=Plur	18	nsubj	18:nsubj	_
```

Comment:
shouldn't the lemma be capitalized here? DM

Reply:
I can believe that, but just a few words earlier it was `dm` not `DM`.  Basically we need to pick a convention and stick with it

Reply:
Yep. I think we went through and fixed the lemmas for some of the more frequent acronyms but missed some of the rarer ones.

Reply:
Maybe mark this in our heads as something that belongs in a separate change?  `id`, `ngo`, `dm`, `cd`, and also in the other `cd` sentence, `lp`

I do agree that every time you see `ID` in the wild, it's written with caps, so likely it should stay that way in the lemma form